sorc/global_chgres.fd/chgres_utils.f90

```diff
@@ -536,8 +536,8 @@ SUBROUTINE VINTG(IM,IX,KM1,KM2,NT,P1,U1,V1,T1,Q1,W1,P2,           &
 ! - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
 !  COMPUTE LOG PRESSURE INTERPOLATING COORDINATE                        
 !  AND COPY INPUT WIND, TEMPERATURE, HUMIDITY AND OTHER TRACERS         
-!$OMP PARALLEL DO DEFAULT(SHARED)                                       
-!$OMP+ PRIVATE(K,I)                                                     
+!$OMP PARALLEL DO DEFAULT(SHARED), &
```

Comment:
Code changes included in the PR intended for build changes.
Please issue a separate PR for code related changes.

Reply:
> Code changes included in the PR intended for build changes.
> Please issue a separate PR for code related changes.

I can check, but I only intended to change the syntax of the OpenMP directives to satisfy the latest version of the Intel compiler.

Reply:
Some background:  The old build system applied the '-qopenmp' flag to a subset of the global_chgres routines.  For some reason, chgres_utils.f90 was not compiled with that flag.  The CMake build compiles all routines with -qopenmp.  And the compiler complained about the old syntax.

Reply:
Ok. I can understand this change and am willing to slide.

Reply:
> Code changes included in the PR intended for build changes.
> Please issue a separate PR for code related changes.
> 
> I can check, but I only intended to change the syntax of the OpenMP directives to satisfy the latest version of the Intel compiler.

Some follow up:  I ran a canned `global_chgres` case on Dell using 14 threads.  The branch (28a11c9d) and develop (270f9dc) produced identical output (according to the nccmp utility).   So I am confident I did not introduce any errors into the code.